pixi.toml

```diff
@@ -97,6 +98,7 @@ colcon-common-extensions = "*"
 [feature.obelisk.pypi-dependencies]
 mujoco = ">=3.1.0"
 "ruamel.yaml" = ">=0.18.0"
+urdf_parser_py = "*"
```

Comment:
put an explicit version floor (I find this sometimes helps clarify the age of the code years down the line when people are trying to resolve a hairy dependency conflict)